client/my-sites/themes/theme-showcase.scss

```diff
@@ -12,7 +12,6 @@
 		background: var(--studio-gray-0);
 
 		.layout__content {
-			padding: calc(var(--masterbar-height) + 16px) 16px 16px calc(var(--sidebar-width-max));
```

Comment:
This was also overriding other existing styles and causing this large left padding when the toggle-able global sidebar was hidden.